API.md

```diff
@@ -443,7 +443,7 @@ Note, the response will have either uuids or 'GROUP' for the user ids, and a uui
 For descriptions of @msg_from, @msg_to and @ru see messages get
 [Get Message List](#get-message-list)
 
-## Get Conversations count  
+h2: ## Get Conversations count  
```

Comment:
Also need to remove the h2 ^^

Reply:
It isnt there ...... The md viewer is adding it